### PR TITLE
Support pkg-config

### PR DESCRIPTION
**Pull request by loop0day**
Improved some code to support pkg-config.

### CODE REVIEW

File: libglog.pc.in [hunk 1]
```diff
@@ -1,10 +1,10 @@
-prefix=@prefix@
-exec_prefix=@exec_prefix@
-libdir=@libdir@
-includedir=@includedir@
+prefix=@CMAKE_INSTALL_PREFIX@
```
Comment (sergiud) [inline libglog.pc.in]: Doesn't this break autoconf support?

Comment (loop0day) [reply libglog.pc.in]: Maybe it's OK now.

Comment (loop0day) [reply libglog.pc.in]: CI failed because of the lack of <sys/wait.h> in MinGW toolchain.

**Comment by sergiud** [comment]
Please rebase and let the CI run again.

**Pull request closed without merging**